### PR TITLE
Remove Bento unused actions in request controller

### PR DESCRIPTION
**Pull request by eduardoj**
As @dmarcoux [pointed out,](https://github.com/openSUSE/open-build-service/pull/8325#issuecomment-530779012) some actions were not removed from the request controller, those ending in _dialog.

### CODE REVIEW

**Comment by hennevogel** [comment]
@eduardoj have you actually tried to go to this page and look at it? :-)

Catching exceptions for downloading binaries was [removed](https://github.com/openSUSE/open-build-service/commit/4099cbf46ab836e96c33b497de77c49abf8ca936#diff-8b28dc09afdb448acc298de6d8269166L59). No idea why only your PR hits this, the spider test isn't very deterministic...

**Comment by coolo** [comment]
run_as_admin throwing authentication_required is strange as well btw

File: src/api/test/functional/webui/spider_test.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def getlinks(baseuri, body)
       next unless link.port == baseuri.port
       link = link.to_s
       next if link =~ %r{/mini-profiler-resources}
+      # we do not really serve binary packages in the test environment
+      next if link =~ %r{/package/binary/download/*}
```
Comment (coolo) [inline src/api/test/functional/webui/spider_test.rb]: https://www.regular-expressions.info/quickstart.html - just saying

Comment (hennevogel) [reply src/api/test/functional/webui/spider_test.rb]: But it was like that on ~~stackoverflow~~ OBS codebase!

Comment (coolo) [reply src/api/test/functional/webui/spider_test.rb]: el viejo grancanario ... :)

**Comment by hennevogel** [comment]
coverage/coveralls is hanging. see [codecov](https://codecov.io/github/eduardoj/open-build-service/commit/e275d9b2fcf6acafe5c17029071747bd816903b8). merging.

**Pull request merged**